apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/User/UserOverviewItemNode.cs

```diff
@@ -102,4 +69,10 @@ public async Task<CollectionSegment<UserOverviewItemDto>> GetUsersAsync(
             pageInfo,
             totalCount);
     }
+
+    #region Computed fields on UserOverviewItemDto
+
+    public static string GetName(
+        [Parent] UserOverviewItemDto user) => user.FirstName + ' ' + user.LastName;
+    #endregion
```

Comment:
praise: interesting take on a way to improve readability, let's see how it feels